site/_docs/configuration.md

```diff
@@ -32,7 +32,8 @@ class="flag">flags</code> (specified on the command-line) that control them.
     <tr class="setting">
       <td>
         <p class="name"><strong>Site Source</strong></p>
-        <p class="description">Change the directory where Jekyll will read files</p>
+        <p class="description">Change the directory where Jekyll will
+        read files</p>
```

Comment:
Paragraphs are a block tag, there must *always* be a new line after the `<p>` just so you know, those are just the unwritten style rules for HTML.  If it's a block level element imply it's block level by giving it a new line, if it's inline then imply it's inline without a new line.